src/global_module.ts

```diff
@@ -0,0 +1,35 @@
+import { readJson, writeJson } from "../deps.ts";
+
+export interface GlobalModule {
+  registry: string;
+  moduleName: string // TODO(@qu4k): ugly
```

Comment:
Is this meant to be completed in this PR?

Reply:
Later, it needs to be changed in the update handler as well if you want to change it @qu4k

Reply:
part of the code cleaning, put in place to see if this style fits